04.ls/ls.rb

```diff
@@ -2,15 +2,48 @@
 # frozen_string_literal: true
 
 require 'optparse'
+require 'etc'
+
+FILE_TYPE = {
```

Comment:
```suggestion
FILE_TYPES = {
```

ハッシュは複数形、単数形、どっちもありますが、他が複数形なら複数形で統一した方がいいかも

Reply:
複数形で統一しました。